src/Inkluzitron/Services/UsersService.cs

```diff
@@ -17,6 +17,16 @@ public UsersService(BotSettings botSettings, DiscordSocketClient discordClient)
             DiscordClient = discordClient;
         }
 
+        public async Task<SocketGuildUser> GetUserFromHomeGuild(ulong userId)
+        {
+            var homeGuild = DiscordClient.GetGuild(BotSettings.HomeGuildId);
```

Comment:
Nemůže nastat situace, že by to HomeGuild nenašlo? Třeba v DEV

Reply:
Může, tak to spadne :pepela:

Alternativa by byla napsat warning do konzole, že není nastavený homeguild. Ale to je moc práce a jednodušší je že to spadne, když se použije nějaký homeguild závislý příkaz.